lib/functions/detectFiles.sh

```diff
@@ -504,12 +504,17 @@ function RunAdditionalInstalls() {
   if [ "${VALIDATE_TERRAFORM_TFLINT}" == "true" ] && [ "${#FILE_ARRAY_TERRAFORM_TFLINT[@]}" -ne 0 ]; then
     info "Detected TFLint Language files to lint."
     info "Trying to install the TFLint init inside:[${WORKSPACE_PATH}]"
+    # Set the log level
+    TF_LOG_LEVEL="info"
+    if [ "${ACTIONS_RUNNER_DEBUG}" = "true" ]; then
+      TF_LOG_LEVEL="debug"
+    fi
     #########################
     # Run the build command #
     #########################
     BUILD_CMD=$(
       cd "${WORKSPACE_PATH}" || exit 0
-      tflint --init 2>&1
+      "tflint --init --loglevel=${TF_LOG_LEVEL}" 2>&1
```

Comment:
I feel like `tflint --init` will always be a no-op atm. 
`tflint --init` is used to install plugins from the config file and we are not providing a config file.
AWS plugin work because it is manually installed. `tflint --init` will only do something if it has something to download like :
plugin "foo" {
  enabled = true
  version = "0.1.0"
  source  = "github.com/org/tflint-ruleset-foo"
}

To make it work we need to point to the config file I believe:
```suggestion
      tflint --init --loglevel="${TF_LOG_LEVEL}"  -c "${TERRAFORM_TFLINT_LINTER_RULES}" 2>&1
```